src/createAnimatedComponent.js

```diff
@@ -229,7 +229,8 @@ export default function createAnimatedComponent(Component) {
     }
 
     _updateFromNative(props) {
-      this._component.setNativeProps(props);
+      // eslint-disable-next-line
```

Comment:
Could you disable a specific warning and write a comment about why it's ignored?

Reply:
fixed 65f7cfb